tools/turbinia_config_parser.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# # Copyright 2016 Google Inc.
+# #
+# # Licensed under the Apache License, Version 2.0 (the "License");
+# # you may not use this file except in compliance with the License.
+# # You may obtain a copy of the License at
+# #
+# #      http://www.apache.org/licenses/LICENSE-2.0
+# #
+# # Unless required by applicable law or agreed to in writing, software
+# # distributed under the License is distributed on an "AS IS" BASIS,
+# # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# # See the License for the specific language governing permissions and
+# # limitations under the License.
+
+
+"""Turbinia Config Parser."""
+
+from __future__ import unicode_literals
+import imp
+import itertools
+import os
+import sys
+
+# This Turbinia config parser can be invoked by any scripts to get the
+# key:value from the Turbinia config file. Note that this will not look
+# for config files anywhere, except for /etc/turbinia. If there's a need
+# to run Turbinia based on the config file in a homedir, you will need
+# to run manually Turbinia.
```

Comment:
Maybe suggest symlinking to the home directory instead of a manual run?

Also, just a nit, but I would say "manually run" rather than "run manually" (in case that sentence stays).

Reply:
Done